src/protocolluaapi.cpp

```diff
@@ -182,9 +182,9 @@ void ProtocolLuaApi::parseHttpRequest(std::string& appName, std::string& name, N
 	/* We are searching if we can even find a registered GlobalEvent with the name
 	*  This way we could push app names onward from the API to only execute the GlobalEvent which belongs to this app
 	*/
-	auto http = g_globalEvents->getEventMap(GLOBALEVENT_HTTPREQUEST);
+	auto& http = g_globalEvents->getEventMap(GLOBALEVENT_HTTPREQUEST);
 	if (http.find(name) != http.end()) {
-		g_globalEvents->getEventMap(GLOBALEVENT_HTTPREQUEST).at(name).executeHttpRequest(name, dataMap);
```

Comment:
Don't call `find` twice, reuse the iterator instead.

Reply:
I guess you probably commented the moment I've added the commit, as this was already resolved ^^